### DISCUSSION TITLE
/asrock-rack-e3c246d4u-the-perfect-media-server-motherboard/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#189] [Announcements]
ASRock Rack E3C246D4U - The Perfect Media Server Motherboard?
        
[https://blog.ktz.me/asrock-rack-e3c246d4u-the-perfect-media-server-motherboard/](https://blog.ktz.me/asrock-rack-e3c246d4u-the-perfect-media-server-motherboard/)

**Comment by migratecomments** [comment]
```header
Originally posted by Benjamin Weiss on 2022-02-03 13:02
```

I'm sorry for what is probably a stupid question, but it's been a couple of decades since I built a custom rig. I'm wanting to build a TrueNAS SCALE system with 8 HDD for storage and an NVME for OS boot. It wasn't clear from your article if you lose SATA 0 if you put in an NVME drive (or in your article, the hyper card. If I put in the NVME M.2 boot device, do I lose a SATA HDD? Thank you!

**Reply by migratecomments** [reply]
```header
Originally posted by Raul Ruiz on 2022-11-13 01:42
```

You lose SATA port 0, there are plenty of SATA ports in this motherboard (from SATA 1 to SATA 7)

**Comment by Forge64** [comment]
You only lose use of a SATA port if you install a SATA M.2 drive. It has to logically route it somewhere. If you install an NVME M.2, you can use all eight SATA ports.

**Reply by ironicbadger** [reply]
My god this explains it. I don't think I ever got my head around until now. Thanks for the comment.

**Comment by roedesh** [comment]
I just want to say that is blog post is an absolute gem! 

I have this exact motherboard and Xeon E-2104G CPU in my new NAS/server build and I was struggling to get the iGPU to show up in Proxmox. After enabling the "IGPU Multi-Monitor" setting, Proxmox was finally able to see my iGPU. I would have never found those hidden settings if it wasn't for this blog post.

The iGPU is now being passed through to my Docker LXC, which is then mounted by a Docker container running Jellyfin. The hardware transcoding works great.

Thank you!

**Reply by ironicbadger** [reply]
They hide it really well don't they?! Glad you found some utility here. 👍

**Reply by saxon-viking** [reply]
I've finally got my new server built (not migrated the DAS drives yet) and it's all NVME and SSD's, and after checking quiet cooling options, I bought an AIO liquid cooling setup. What a huge speed increase over my existing setup. I love the IPMI as well. Will be working on the iGPU and QSV for my Jellyfin now.....
Just got to work out what I'm going to do with the 4 NVME's now! hahaha probably cache for the hdd's.
Thanks for the great info Alex!

**Comment by jorgegarciaavila** [comment]
Hello Alex, what LSI card and HDD did you use for Morpheus?, been watching your videos rebuilding it but didn't paid attention to the details and I just started planning on using a similar case/asrock mobo for a nas, I think i have everything figured out, will use an E3C246D4U2-2T with an E-2124G, 32 GB ram, the ASUS 4xNVME card (using only 3), and havent decided if going for a LSI or just add a 6 SATA X4 pcie card, will be using proxmox, zfs by host and bind mouting, already running it like that and runs really well as it can access all of the RAM, thanks and cheerio

**Comment by AngusMacGyver-DE** [comment]
Hi Alex,

unfortunately the ASRock Rack E3C246D4U is quite impossible to find theese days.
I have already seen your Video with your Upgrade to a Supermicro X11SAE-F but this to big for my build.

I wonder if you could recommend and/or know a board in Micro-ATX form factor with both IPMI and iGPU usability with decent pricing and availability theese days?

Thank you for any hints!

**Comment by saxon-viking** [comment]
Hi all,

I have this motherboard E3C246D4U2-2T, this one has 2 x 10Gbe LAN ports (only found out they were 10Gbe last week by accident!). I have found a quirk with the BIOS. I hadn't connected my server back to a UPS and while I was out, my wife had electricians in the house. Not aware of my server, the electricians doing their stuff upgrading incoming fuses from 60 to 100amps, went off / on / off / on / off / on while doing checks. So did my server, which messed it up but very subtly. I found that the 4 x NVMe drives (same set up as you Alex) had gone degraded. Mine is set up 2 x NVMe as tank1, 2 x NVMe as tank2, both mirrored, and T2 is an exact duplicate of T1.

T2 went degraded, but I could still access the datasets. I searched far and wide on various forums, Google, Whoogle, AI's, none of them gave me the answer to fix the problem. Nothing worked. So after a few weeks, I rebooted through IPMI and thought I'd check the settings there. Well, what I found was that the bifurcation for 8x4x4 had changed to AUTO. So this had (didn't look too hard) but it had removed one of the NVMe's from the PCIe lanes.

So moral of the story, use a UPS, secondly, if you get degraded NVME's and can't find what it is, check that your bios hasn't reset itself to AUTO on the PCIe settings.

Hope that helps someone :-D
